tests/pkg-rendition-paginated/OPF/css/base.css

```diff
@@ -0,0 +1,94 @@
+@charset "utf-8";
+
+body {
+    font-family: arial, helvetica, sans-serif;
+    font-size: 1em;
+    color: black;
+    background: white;
+}
+
+h1 { color: rgb(0,90,156); }
+h2 { font-size: 140%; }
+h3 { font-size: 130%; }
+h4 { font-size: 120%; }
+
+.ctest h2, .ctest h3, .ctest h4, .ctest h5, .otest h2, .otest h3, .otest h4, .otest h5 {
+	font-size: 120%;
+	color: rgb(0,0,0);
+}
+
+.test-id, .demo-ctest {    
+    font-size: 80%;
+    font-style: normal;    
+    color:gray;
+    margin-right: 1em;    
+    padding: 0.3em;
+}
+
+.ctest, .otest {
+    margin-bottom: 1em;
+    padding-left: 1em;
+}
+
+.ctest {
+    border: 1px solid gray;
+    margin-bottom: 1em;
+    padding-left: 1em;
+}
+
+.otest {
+    border: 1px dotted gray;
+    margin-bottom: 1em;
+    padding-left: 1em;
+}
+
+.nature {
+	display: block;
+	padding-right: 1em;
+	padding-left: 0.5em;
+	font-size: 50%;
+}
+
+.ctest .nature {
+	color: rgb(255,0,0);
+}
+
+.otest .nature {
+	color: rgb(0,0,255);
+}
+
+.test-label-info {
+	display: inline;
+	font-size: 100%
+}
+
+dl.info dt {
+    font-weight: bold;
+    margin-bottom: 1em;
+}
+
+nav#toc ol  {
+    list-style-type: none;
+}
+
+nav a {
+	text-decoration:none;
+	color:black;
+}
+
+.eval {
+	font-weight:bold;
+}
+
+.eval:before {
+	content:"→";
+}
+
+.conditional {
+	font-style: italic;
+}
+
+
```

Comment:
Actually... it goes deeper than that. I neglected to adapt the tests to the general style of our tests, which I have now done. This meant:

- removing the css file altogether (not necessary for the tests that, we agreed, should be minimal in term of features)
- changing the top level `OPF` directory to `OPS`
- changing the naming schemes of the XHTML content (instead of `xhtml/xhtml-00*` to `content_00*`

All the tests have been re-generated accordingly...